samples/app-identity-linking-with-sso/csharp/IdentityLinkingWithSSO/Pages/tab.cshtml

```diff
@@ -138,14 +134,15 @@
                     }
                     else {
                         const userDetails = JSON.parse(responseJson);
+                        console.log(userDetails);
                         if (userDetails.isAadSignedIn) {
                             if (userDetails.isGoogleSignedIn) {
                                 is_google_signed_in = true;
-                                setTimeout(() => { getGoogleDetails(userDetails.GoogleToken) }, 3000);
+                                setTimeout(() => { getGoogleDetails() }, 3000);
```

Comment:
getGoogleDetails rename it as getGoogleAccountDetails or getGoogleAccountUserDetails